selfdrive/car/toyota/interface.py

```diff
@@ -233,8 +242,8 @@ def update(self, c):
     ret.cruiseState.speed = self.CS.v_cruise_pcm * CV.KPH_TO_MS
     ret.cruiseState.available = bool(self.CS.main_on)
     ret.cruiseState.speedOffset = 0.
-    if self.CP.carFingerprint == CAR.RAV4H:
-      # ignore standstill in hybrid rav4, since pcm allows to restart without
+    if (self.CP.carFingerprint == CAR.RAV4H) or (self.CP.carFingerprint == CAR.HIGHLANDERH):
```

Comment:
`if self.CP.carFingerprint in [CAR.RAV4H, CAR.HIGHLANDERH]:`

Reply:
Finished touch up!